src/plugins/sqlite.js

```diff
@@ -35,7 +35,7 @@ class MessageStorage {
 
 		this.isEnabled = true;
 
-		this.database = new sqlite3.cached.Database(sqlitePath);
```

Comment:
Caching doesn't do anything for us. (all it does is keeps internal reference for the filename).